nl_server/loader.py

```diff
@@ -121,12 +128,15 @@ def _load_yaml(flask_env: str) -> Dict[str, str]:
   return embeddings_map
 
 
-def _update_app_config(app: Flask, nl_model: NLAttributeModel,
+def _update_app_config(app: Flask,
+                       nl_model: NLAttributeModel,
```

Comment:
find nl_model and vertex_ai_models a bit confusing. I feel either have one model class to represent both or to make the nl_model more specific.

Reply:
it looks like nl_model is used for detecting verbs, maybe I can rename it to nl_attribute_model?

Reply:
maybe we can remove "nl" (which applies to everything) and use "attribute_model"?